packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -113,6 +134,16 @@ - (void)updateJsMode:(NSNumber*)mode {
   }
 }
 
+- (void)updateUserAgent:(NSString*)userAgent {
+  if (@available(iOS 9.0, *)) {
+    [_webView setCustomUserAgent:userAgent];
+  } else if (userAgent) {
+    [[NSUserDefaults standardUserDefaults] registerDefaults:@{@"UserAgent" : userAgent}];
```

Comment:
Why is this required? Where is the value read again?

Reply:
Looks like `WkWebView` reads that value on construction, I did not find official documentation for this other than SO, but I did verify that it works.

Reply:
In that case.. I’d recommend a short source code comment, perhaps a link to the relevant SO answer.